MultiplayerSample_Dependencies.xml

```diff
@@ -1,6 +1,22 @@
 <EngineDependencies versionnumber="1.0.0">
     <Dependency path="*.azshadervariant*" optional="false" /> <!-- The wildcard includes azshadervarianttree assets too -->
```

Comment:
Actually, isn't this line broken too? The azshadervariant files exist in multiple subdirectories.

Reply:
I'll double check when this build finishes, but I thought that was matching differently because there were no directory markers, so it was just matching on file name.

Reply:
It looks like it's picking them up. I'm seeing a lot of shader variants:

![image](https://user-images.githubusercontent.com/4838196/228973289-aad3d955-e149-4b22-9665-b6014787dc93.png)

The scan folder path to that first one is:
Shaders/DiffuseGlobalIllumination/diffuseprobegridvisualizationraytracingmiss_dx12_0.azshadervariant